core/src/main/java/com/linecorp/armeria/client/retry/RetryingClientBuilder.java

```diff
@@ -72,6 +73,22 @@ public T defaultMaxAttempts(int defaultMaxAttempts) {
     }
 
     /**
+     * Sets the {@code responseTimeoutMillisForEachAttempt}. When requests in {@link RetryingClient} are made,
+     * corresponding responses are timed out by this value. {@code 0} disables the timeout.
+     * It will be set by the default value in {@link Flags#defaultResponseTimeoutMillis()}, if the client
+     * dose not specify.
+     *
+     * @return {@link T} to support method chaining.
+     */
+    public T responseTimeoutMillisForEachAttempt(long responseTimeoutMillisForEachAttempt) {
```

Comment:
How about adding `responseTimeoutForEachAttempt(Duration)` as well, which delegates to this method?

Reply:
Added~!! Thank you~!